### PR TITLE
FE-1395: Fix Codescan problems

### PR DESCRIPTION
**Pull request by kchu1337**
https://github.com/SparkPost/tools-ui/security/code-scanning?query=tool%3ACodeQL+ref%3Arefs%2Fpull%2F71%2Fmerge

What Changed:
Addressed all codescan issues

1. Removed conditional webpack/babel paths that didn't seem to be used
2. Sanitized a link.
3. Renamed auth token key from 'Authorization' to 'authHeader'
4. Removed empty test
5. Removed unnecessary imports

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -158,72 +158,25 @@ function build(previousSizeMap) {
     console.log();
 
     var openCommand = process.platform === 'win32' ? 'start' : 'open';
```
Comment (kchu1337) [inline scripts/build.js:160]: Removed unnecessary conditional paths based on https://travis-ci.org/github/SparkPost/tools-ui/jobs/761625807

File: src/helpers/http.js [hunk 2]
```diff
@@ -13,7 +13,10 @@ function useRefreshToken(refresh_token) {
     method: 'POST',
     url: '/authenticate',
     data: `grant_type=refresh_token&refresh_token=${refresh_token}`,
-    headers: authCookie.requestHeaders
+    headers: {
```
Comment (kchu1337) [inline src/helpers/http.js:16]: This is similar to the way we do it in 2web2ui

**Review by PatrickSison — COMMENTED** [review]
I think it looks good. I don't think I saw where the variable got sourced for `appPackage.homepage` in the package.json anyways. I also wasn't sure if there needed to be a test around the ResultsError or we were just cool with deleting. Let me know about the deleted test or if we're good to drop and then we can move forward with this ticket

**Review by PatrickSison — APPROVED** [review]

**Comment by kchu1337** [comment]
> I think it looks good. I don't think I saw where the variable got sourced for `appPackage.homepage` in the package.json anyways. I also wasn't sure if there needed to be a test around the ResultsError or we were just cool with deleting. Let me know about the deleted test or if we're good to drop and then we can move forward with this ticket

I feel like it's fine to remove since it wasn't doing anything in the first place and I didn't change any functionality of the component. In addition, we are probably never going to touch this repo again.

**Pull request merged**